java/src/jmri/jmrit/symbolicprog/tabbedframe/PaneProgPane.java

```diff
@@ -1627,7 +1639,14 @@ protected void addListener(java.beans.PropertyChangeListener qc) {
     }
 
     /**
-     * Create a new grid group from the JDOM group Element
+     * Create a new grid group from the JDOM group Element.
+     *
+     * @param element     element containing group contents
+     * @param c           the panel to create the grid in
+     * @param g           the layout manager for the panel
+     * @param globs       ???
```

Comment:
@rhwood

> @dheap What are the GridGlobals and how are they used?

GridGlobals are per-grid class instances (containers) that hold:
- The current (last-used) x and y grid positions for that grid. Used by the CURRENT and NEXT constants for "gridx" and "gridy" attributes.
- Any default attribute values specified for this particular grid in the <grid> element. Attributes  are overridable in any particular <griditem> element.
- The GridBagConstraints class instance for that particular grid panel.

http://jmri.org/help/en/html/apps/DecoderPro/Programmer.shtml#grid


> What is the showStdName parameter?

This is the 'nameFmt' attribute of a variable from a pane definition file as per
http://jmri.org/help/en/html/apps/DecoderPro/Programmer.shtml#display